identities/tasks.py

```diff
@@ -21,3 +24,23 @@ def run(self, anum, **kwargs):
         return int(anum)+1
 
 the_incr = The_Incr()
+
+
+class DeliverHook(Task):
+    def run(self, target, payload, instance=None, hook=None, **kwargs):
+        """
+        target:     the url to receive the payload.
+        payload:    a python primitive data structure
+        instance:   a possibly null "trigger" instance
+        hook:       the defining Hook object
+        """
+        requests.post(
+            url=target,
+            data=json.dumps(payload),
+            headers={
+                'Content-Type': 'application/json',
+                'Authorization': 'Token %s' % settings.HOOK_AUTH_TOKEN
```

Comment:
@hodgestar Clarification check: so when each service registers a hook it should also provide a token that the Identity Store can use to authenticate with the service?